pkg/helmexec/exec.go

```diff
@@ -145,10 +145,16 @@ func (helm *execer) DecryptSecret(context HelmContext, name string, flags ...str
 	}
 	defer tmpFile.Close()
 
+	// HELM_SECRETS_DEC_SUFFIX is used by the helm-secrets plugin to define the output file
+	decSuffix := os.Getenv("HELM_SECRETS_DEC_SUFFIX")
+	if len(decSuffix) == 0 {
+		decSuffix = ".yaml.dec"
+	}
+	decFilename := strings.Replace(name, ".yaml", decSuffix, 1)
```

Comment:
This seems to break when the file ext. is something other than `.yaml`.
For example `foo.secrets.yml` would remain `foo.secrets.yml` regardless of `HELM_SECRETS_DEC_SUFFIX` which isn't what you would want:

How about trimming the trailing `.yaml`, and then adding the decSuffix?

Perhaps:

```
decFilename := strings.TrimSuffix(name, ".yaml") + decSuffix
```

Reply:
Well, in fact the helm-secrets plugin doesn't support any other extension than `.yaml` - see https://github.com/futuresimple/helm-secrets/blob/master/secrets.sh#L306

```
__ymldec=$(sed -e "s/\\.yaml$/${DEC_SUFFIX}/" <<<"$yml")
```

so anyway a `.yml` extension will never be decrypted.
and if we come up with a different naming for the decrypted file, it will just not exists.

Reply:
Good to know! Thanks for confirming ☺️